multimodal/src/autogluon/multimodal/utils/object_detection_visualizer.py

```diff
@@ -0,0 +1,493 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# Disclaimer: Special thanks to the Detectron2 developers
+# https://github.com/facebookresearch/detectron2/blob/main/detectron2/utils/visualizer.py!
+# We use part of its provided, open-source functionalities.
+
+import colorsys
```

Comment:
Is this (`colorsys`) something we need to put it into dependencies?

Reply:
should be a default package: https://docs.python.org/3/library/colorsys.html